test/blackbox-tests/test-cases/pkg/lock-per-context.t/run.t

```diff
@@ -7,8 +7,14 @@ Create a workspace with multiple contexts, each specifying a lockdir name.
   > (context
   >  (default
   >   (name foo)
+  >   (prefer_versions newest) ; this is the default
   >   (lock bar.lock)))
   > (context
+  >  (default
+  >   (name prefers_oldest)
+  >   (prefer_versions oldest)
```

Comment:
Eventually, we'll want a field to group all solver settings. But this is fine for now.